manim/mobject/svg/text_mobject.py

```diff
@@ -1361,7 +1361,7 @@ def extract_gradient_tags(self):
         Removes the ``<gradient>`` tag, as it is not part of Pango's markup and would cause an error.
         """
         tags = re.finditer(
-            '<gradient\s+from="([^"]+)"\s+to="([^"]+)"(\s+offset="([^"]+)")?>(.+?)</gradient>',
+            r'<gradient\s+from="([^"]+)"\s+to="([^"]+)"(\s+offset="([^"]+)")?>(.+?)</gradient>',
```

Comment:
cc @naveen521kk @PhilippImhof: is there a reason why these strings haven't been raw strings before?

Reply:
All I know is it was causing a warning in tests. Somewhat related to #1149

Reply:
> is there a reason why these strings haven't been raw strings before?

Something which was missed probably.